code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -16,18 +16,20 @@
 	layer = 2.9
 	clicksound = /singleton/sound_category/button_sound
 
-	var/beaker = null
+	var/obj/item/reagent_containers/glass/beaker = null
 	var/obj/item/storage/pill_bottle/loaded_pill_bottle = null
 	var/mode = TRUE
 	var/condi = 0
```

Comment:
If it's a condimaster, make it have the idris background instead of the zenghu one.

Reply:
done